public/app/features/scenes/components/layout/SceneGridLayout.tsx

```diff
@@ -0,0 +1,574 @@
+          x: size.x!,
+          y: size.y!,
+          w: size.width!,
+          h: size.height!,
+          isResizable,
+          isDraggable,
+        });
+      } else {
+        // Rendering row children
+        let isDraggable = Boolean(child.child.state.isDraggable);
+        let isResizable = Boolean(child.child.state.isResizable);
+        const cell = {
+          i: child.child.state.key!,
+          x: size.x!,
+          // y: size.y!,
+          y: size.y! + child.row.state.size!.y!,
+          w: size.width!,
+          h: size.height!,
+          isResizable,
+          isDraggable,
+        };
+        if (perRowCells[child.row.state.key!]) {
+          perRowCells[child.row.state.key!].push(cell);
+        } else {
+          perRowCells[child.row.state.key!] = [cell];
+        }
+      }
+    });
+
+    topLevelCells = topLevelCells.sort((panelA, panelB) => {
+      if (panelA.y === panelB.y) {
+        return panelA.x - panelB.x;
+      } else {
+        return panelA.y - panelB.y;
+      }
+    });
+    // sort sub row cells
+    for (const rowKey in perRowCells) {
+      perRowCells[rowKey] = perRowCells[rowKey].sort((panelA, panelB) => {
+        if (panelA.y === panelB.y) {
+          return panelA.x - panelB.x;
+        } else {
+          return panelA.y - panelB.y;
+        }
+      });
+    }
+
+    const lg = topLevelCells.reduce<ReactGridLayout.Layout[]>((acc, cell) => {
+      if (perRowCells[cell.i]) {
+        return [...acc, cell, ...perRowCells[cell.i]];
+      }
+      return [...acc, cell];
+    }, []);
+
+    return { lg, sm: lg.map((l) => ({ ...l, w: 24 })) };
+  }, [model.flattenedChildren]);
+
+  return (
+    <AutoSizer disableHeight>
+      {({ width }) => {
+        if (width === 0) {
+          return null;
+        }
+
+        // Dev only, to be removed
+        const background = generateGridBackground({
+          cellSize: { width: (width - 23 * GRID_CELL_VMARGIN) / 24, height: GRID_CELL_HEIGHT },
+          margin: [GRID_CELL_VMARGIN, GRID_CELL_VMARGIN],
+          cols: 24,
+          gridWidth: width,
+          theme,
+        });
+
+        return (
+          /**
+           * The children is using a width of 100% so we need to guarantee that it is wrapped
+           * in an element that has the calculated size given by the AutoSizer. The AutoSizer
+           * has a width of 0 and will let its content overflow its div.
+           */
+          <div style={{ width: `${width}px`, height: '100%', background }}>
+            <ReactGridLayout
+              width={width}
+              /*
+                  Disable draggable if mobile device, solving an issue with unintentionally
+                  moving panels. https://github.com/grafana/grafana/issues/18497
+                  theme.breakpoints.md = 769
+                */
+              isDraggable={width > 768}
+              isResizable={false}
+              containerPadding={[0, 0]}
+              useCSSTransforms={true}
```

Comment:
this is not working, there is no movement animation (think it could be disabled via css in _dashboard_grid.scss